terragrunt/aws/api/inputs.tf

```diff
@@ -0,0 +1,31 @@
+variable "api_auth_token" {
+  description = "The API token needed for all requests to the API."
+  type        = string
+  sensitive   = true
+}
+
+variable "url_shortener_table_name" {
+  description = "DynamoDB table name for the URL shortener"
+  type        = string
+}
+
+variable "ecr_repository_arn" {
+  description = "Arn of the ECR Repository"
+  type        = string
+}
+
+variable "ecr_repository_url" {
+  description = "URL of the URL shortener ECR"
+  type        = string
+}
+
+variable "api_id" {
```

Comment:
Minor comment - could you rename this to something like `api_security_group_id` just so it's clearer.

Reply:
Done!